packages/components/src/components/search/search.tsx

```diff
@@ -233,6 +238,7 @@ export class Search implements ComponentInterface {
                 this.search(inputValue);
             }
         }
+        this.changed.emit();
```

Comment:
Should this be moved inside the `if (!this.preventSearch)` statement? And I'm concerned about the return statement within that too. Do we risk not emitting when we should have?

Reply:
I moved it inside the statement. What would be the case when we would not emit when we should?

Reply:
From what I can see, we will not emit if the length of the input is less than two characters (due to the return). Should we emit in this case?

Reply:
Hmm, I think that is okay, isn't it? Because if there are less than two characters, we are emitting the shortInput event and we can use that if we need. Do you think we should emit the event in that case as well?